src/styles/blocks/search-bar__input.css

```diff
@@ -0,0 +1,21 @@
+.search-bar__input {
+  width: 100%;
+  padding: 0;
+  border: 0;
+
+  font-family: Avenir, Arial, 'Helvetica Neue', Helvetica, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. This ensures that if the primary font fails to load for some reason, the browser has alternative fonts to fall back on. For example, you could specify a generic font family as a fallback: `font-family: Avenir, Helvetica, Arial, sans-serif;`